### ISSUE TITLE
rhs -> rnf

### ISSUE THREAD

**Issue by LeventErkok** [ndmitchell/hlint#625]
There's a typo in:

https://github.com/ndmitchell/hlint/blob/master/data/hlint.yaml#L439

which causes this program:

```haskell
foo = rnf 4 `seq` ()
```

to produce

```
$ hlint a.hs
a.hs:1:7: Warning: Redundant seq
Found:
  rnf 4 `seq` ()
Perhaps:
  rhs 4

1 hint
```

Should suggest:

```
   rnf 4
```

instead. (Note the `rhs`/`rnf` confusion.)
